### PR TITLE
Add help for --with-service-type command option in OpenAPI tool

### PR DESCRIPTION
**Pull request by aneeshafedo**
Documentation for the fix https://github.com/ballerina-platform/openapi-tools/issues/1211

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help [hunk 1]
```diff
@@ -82,6 +82,9 @@ OPTIONS
         --client-methods <resource|remote>
             This option can be used in client generation to select the client method type, which can be `resource` or `remote`. (The default option is `remote`).
 
+        --with-service-type
+            This option can be used in service generation to generate the service with the service type object.
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help:86]:
```suggestion
            This option can be used to generate the service with a service-type object.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help [hunk 2]
```diff
@@ -118,6 +121,10 @@ EXAMPLES
            $ bal openapi -i hello.yaml --client-methods resource
            $ bal openapi -i hello.yaml --mode client --client-methods resource
 
+       Generate a Ballerina service with the service type object.
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help:124]:
```suggestion
       Generate a Ballerina service with a service-type object.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help (hunk 1 above)
Comment (aneeshafedo) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help]: Resolved

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help (hunk 2 above)
Comment (aneeshafedo) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help]: Resolved

**Review by praneesha — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

**Review by lnash94 — APPROVED** [review]

**Pull request closed without merging**